Library/Homebrew/formula_installer.rb

```diff
@@ -1152,7 +1152,7 @@ def pour
     tab.arch = Hardware::CPU.arch
     tab.source["versions"]["stable"] = formula.stable.version.to_s
     tab.source["path"] = formula.specified_path.to_s
-    tab.source["tap_git_head"] = formula.tap&.git_head
+    tab.source["tap_git_head"] = formula.tap&.git_head if formula.tap.installed?
```

Comment:
```suggestion
    tab.source["tap_git_head"] = formula.tap&.git_head if formula.tap&.installed?
```

because the formula may not have a tap at all.